supabase/functions/oauth/encrypt-config.ts

```diff
@@ -48,19 +50,23 @@ export async function encryptConfig(req: Record<string, any>) {
   });
 
   let responseData = JSON.stringify(await response.json());
```

Comment:
I realized while reviewing that we can only return the response from `config-encryption` if it has a 200 status. If the response has a 400 status, then it could be due to a JSON schema validation error. In that case, the response body will include the input document as part of the response, which would include the client_secret. I think, as written, this would actually be caught by the check below. But it means that a validation error would be turned into an `exposed_secret` error, which would be pretty confusing.